apis/v1alpha1/generator.yaml

```diff
@@ -183,6 +184,31 @@ resources:
         from:
           operation: RestoreDBInstanceFromDBSnapshot
           path: UseDefaultProcessorFeatures
+      # Used by create db instance read replica
+      SourceDBInstanceIdentifier:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: SourceDBInstanceIdentifier
+      DestinationRegion:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: DestinationRegion
+      ReplicaMode:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: ReplicaMode
+      SourceRegion:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: SourceRegion
+      PreSignedUrl:
+        from:
+          operation: CreateDBInstanceReadReplica
+          path: PreSignedUrl
+      EnablePerformanceInsights:
```

Comment:
`PerformanceInsightsEnabled` already exists on `DBInstance`? Why do we need this as well?

Reply:
I think it's because in DBInstance, `EnablePerformanceInsights` is already renamed to `PerformanceInsightsEnabled`,  controller will throw exception if i don't add it here. 


```
# github.com/aws-controllers-k8s/rds-controller/pkg/resource/db_instance
pkg/resource/db_instance/sdk.go:3684:15: r.ko.Spec.EnablePerformanceInsights undefined (type "github.com/aws-controllers-k8s/rds-controller/apis/v1alpha1".DBInstanceSpec has no field or method EnablePerformanceInsights)
pkg/resource/db_instance/sdk.go:3685:47: r.ko.Spec.EnablePerformanceInsights undefined (type "github.com/aws-controllers-k8s/rds-controller/apis/v1alpha1".DBInstanceSpec has no field or method EnablePerformanceInsights)

```

Reply:
In the same way you rename the field for the Create call, you should do it with this field.

Reply:
I think i'm following the same pattern for `CreateDBInstance` call as this `CreateDBInstanceReadReplica` call 

I have this rename here https://github.com/aws-controllers-k8s/rds-controller/blob/issue-893/generator.yaml#L220-L222 .

it's strange that if i remove the line 208-211 https://github.com/aws-controllers-k8s/rds-controller/blob/issue-893/generator.yaml#L208-L211.  

generator will throw above exception.